tools/appveyor/requirements.txt

```diff
@@ -0,0 +1,17 @@
+# Fetch numpy and scipy wheels from the sklearn rackspace wheelhouse.
+# Those wheels were generated by @ogrisel by calling `wheel convert` on
+# the binaries from http://www.lfd.uci.edu/~gohlke/pythonlibs/
+# This is a temporary solution. As soon as numpy, scipy and cython provide
+# official wheels for windows we can delete this --find-links line.
+--find-links http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/appveyor_pip_index.html
```

Comment:
@matthew-brett Just wanted to confirm with you whether this is the most current situation?